lib/ansible/plugins/action/include_vars.py

```diff
@@ -159,6 +160,7 @@ def run(self, tmp=None, task_vars=None):
             result['failed'] = failed
             result['message'] = err_msg
 
+        result['ansible_included_var_files'] = self.included_files
```

Comment:
we shoudl actually mention this in the RETURN docs, noting that it is available since version 2.4